Fix U21 matches not being recognised for xp calc

unnecessarydave opened the pull request:
The updated code in function getGameType wasn't catching U21 continental cup matches. Adding the relevant icon names to the CONT_CUPS list fixes this.

<!-- Please review the contributing guidelines before submitting this PR. -->

cpf-fin commented:
Similar tests are "reversed", does `gameIcon.includes(CONT_CUPS)` work? (no need to change `CONT_CUPS`)

unnecessarydave commented:
> Similar tests are "reversed", does `gameIcon.includes(CONT_CUPS)` work? (no need to change `CONT_CUPS`)

Sure, we could do it that way.

`gameIcon.includes(CONT_CUPS)` alone doesn't work, as I think `.includes()` takes a search string, not an array.  But, we could loop through the array, which I admit is better.

I'll see if I can figure out how update the PR.

unnecessarydave committed:
fix U21 matches not being recognised for xp calc

The updated code in function getNTType() wasn't catching U21 continental cup
matches.
